lib/ui/view/plugins/add_plugin_view.dart

```diff
@@ -55,7 +57,7 @@ class AddPluginView extends StatelessWidget {
                         value: model.checked,
                         onChanged: (bool? value) {
                           model.setChecked(value!);
-                          print(value);
+                          log.i(value);
```

Comment:
no need for this log, you can remove it. No logic in the UI

Reply:
Alright